demisto_sdk/commands/content_graph/tests/update_content_graph_test.py

```diff
@@ -206,11 +206,10 @@ def _get_pack_by_id(repository: ContentDTO, pack_id: str) -> Pack:
 
 
 def dump_csv_import_files(csv_files_dir: Path) -> None:
-    for is_docker in [True, False]:
-        import_path = neo4j_service.get_neo4j_import_path(is_docker).as_posix()
-        if Path(import_path).is_dir():
-            shutil.rmtree(import_path)
-            copy_tree(csv_files_dir.as_posix(), import_path)
+    import_path = neo4j_service.get_neo4j_import_path().as_posix()
+    if Path(import_path).is_dir():
```

Comment:
doesn't this deserve a separate test?

Reply:
we just removed the `is_docker` argument. The `demisto-sdk` will figure out by itself where it's running.
However, we need to run in nightly more tests to check the everything is functional running with docker.

We can't add this to demisto-sdk build because CircleCI doesn't support docker-in-docker.